bin/cheritest/cheritest_fault.c

```diff
@@ -261,4 +261,20 @@ test_fault_read_epcc(const struct cheri_test *ctp __unused)
 
 	CHERI_CAP_PRINT(cheri_getepcc());
 }
+
+void
+test_fault_cloadtags_misaligned(const struct cheri_test *ctp __unused)
+{
+	uint64_t tags = 0xDEADFEED;
+	void * __capability p[20] __attribute__((aligned(512))) = { 0 } ;
+	void * __capability c = &p[1];
+
+	__asm__ __volatile__ ("cloadtags %[tags], %[c]"
+				: [tags]"r"(tags)
```

Comment:
I'm surprised this even compiles. `=r`, or maybe even `+r` if you want to ensure the `0xDEADFEED` isn't known statically to be dead.